distribution/docs/src/main/resources/content/_configuring/configuring-idp.adoc

```diff
@@ -88,8 +88,8 @@ With that configuration, any clients that failed to connect using either the SAM
 If you have configured `/search` to use IDP, ensure to select the "External Authentication" checkbox in Search UI standard settings.
 ====
 
-.[[_configuring_idp_sp]]Configuring the Service Providers (SP)
-To configure the IdP client that interacts with the specified Identity Provider also known as the Service Provider,
+.[[_configuring_idp_sp]]Configuring the Service Provider (SP)
+To configure the IdP client (also known as the Service Provider) that interacts with the specified Identity Provider also known as the Service Provider,
```

Comment:
✏️ You still need to remove the old "also known as the Service Provider" at the end of the sentence.